libobs/util/platform.h

```diff
@@ -124,7 +124,9 @@ EXPORT char *os_get_abs_path_ptr(const char *path);
 EXPORT const char *os_get_path_extension(const char *path);
 
 EXPORT bool os_get_emulation_status(void);
-
+#ifdef __APPLE__
+EXPORT bool is_apple_silicon(void);
```

Comment:
I _think_ I wanted it to be available "globally" (that is part of libobs' API so that modules/UI/etc can use it) - got a suggestion for a better place?

It _is_ implemented in the cocoa-specific platform file, but I don't think we have platform-specific headers.